atlp-rwanda/ecommerce-app-predators-fe
# Ch-setup-router-49

### What does this PR do?
This pull request implements the setup of a router using React Router in our client application.

### Description of Task to be completed
- Installed and configured React Router to enable client-side routing within the application.
- Created a dummy product route ("/product-page") and a root route ("/") to test the router configuration.


### How should this be manually tested?
1. Run the application and navigate to the product route ("/product-page") to ensure the product page is displayed correctly.
2. Access the root route ("/") and verify that the homepage content is rendered appropriately.


Screenshots (if appropriate)
N/A
